WTWSMS/history/characters/roman.txt

```diff
@@ -5943,11 +5944,13 @@
 	450.1.1 = { death = yes }
 }
 1300792 = {
-	name = "Marcianus" #PLRE vol. 2 Iulius Agrius Tarrutentius Marcianus 20
+	name = "Tarrutenius Marcianus" #PLRE vol. 2 Iulius Agrius Tarrutenius Marcianus 20
 	# comes Orientis soon after 437
+	dynasty = 8648
 	religion = nicene
-	culture = roman
+	culture = romano_aramean
```

Comment:
Was this changed based upon the PLRE?

Reply:
It was changed based on culture of his father and sister.

Reply:
OK.